src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -209,10 +209,15 @@ const ChatItemMyTextV2 = ({
   }, []);
 
   const handleTextContainerStyle = () => {
+    const isAttachment = attachments.length > 0;
+    const paddingStyle = {
+      paddingLeft: isAttachment ? BUBBLE_LEFT_PADDING_ATTACHMENT : BUBBLE_LEFT_PADDING,
+      paddingRight: isAttachment ? BUBBLE_RIGHT_PADDING_ATTACHMENT : BUBBLE_RIGHT_PADDING
+    };
     if (chatType === SIGNED) {
-      return [styles.containerSigned, styles.textContainer];
+      return [styles.containerSigned, styles.textContainer, paddingStyle];
     }
-    return [styles.containerAnon, styles.textContainer];
+    return [styles.containerAnon, styles.textContainer, paddingStyle];
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `handleTextContainerStyle` has been updated to include conditional padding based on the presence of attachments. This is a good approach as it ensures that the UI adapts to different content types (text vs attachments).
```diff
- const handleTextContainerStyle = () => {
-   if (chatType === SIGNED) {
-     return [styles.containerSigned, styles.textContainer];
-   }
-   return [styles.containerAnon, styles.textContainer];
- };
+ const handleTextContainerStyle = () => {
+   const isAttachment = attachments.length > 0;
+   const paddingStyle = {
+     paddingLeft: isAttachment ? BUBBLE_LEFT_PADDING_ATTACHMENT : BUBBLE_LEFT_PADDING,
+     paddingRight: isAttachment ? BUBBLE_RIGHT_PADDING_ATTACHMENT : BUBBLE_RIGHT_PADDING
+   };
+   if (chatType === SIGNED) {
+     return [styles.containerSigned, styles.textContainer, paddingStyle];
+   }
+   return [styles.containerAnon, styles.textContainer, paddingStyle];
+ };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->